lib/livebook_web/live/session_live/secrets_component.ex

```diff
@@ -219,6 +217,32 @@ defmodule LivebookWeb.SessionLive.SecretsComponent do
     """
   end
 
+  defp prefill_assigns(%{assgins: %{data: %{"name" => _}}} = socket), do: socket
```

Comment:
There is a typo in the assigns key, which means this clause is never called. :) I don't know if we should fix the typo or remove the clause altogether.